pkg/infoschema/infoschema.go

```diff
@@ -308,6 +308,19 @@ func (is *infoSchema) TableByID(_ stdctx.Context, id int64) (val table.Table, ok
 	return slice[idx], true
 }
 
+func (is *infoSchema) SchemaNameByTableID(tableID int64) (schemaName pmodel.CIStr, ok bool) {
+	tbl, ok := is.TableByID(stdctx.Background(), tableID)
```

Comment:
how about pass the ctx instead of the background ctx

Reply:
We don't really use that context.